brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3405 Fix: Post Gird/Carousel - Resolved an issue wherein the grid/carousel showing deprecated `preg_replace` warning.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3405

### Screenshots

- https://d.pr/v/mGhyDf

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Post Grid/Carousel/Timeline on **Post**.
- Publish the Post and visit frontend.
- In debug log as well as in query monitor you will see an PHP issue.
- Switch to mention branch refresh the page. Issue will get resolved.

### Checklist:
